pkg/forwarder/forwarder.go

```diff
@@ -498,6 +499,12 @@ func (f *DefaultForwarder) SubmitV1Series(payload Payloads, extra http.Header) e
 	return f.sendHTTPTransactions(transactions)
 }
 
+// SubmitSeries will send timeseries to the v2 endpoint
+func (f *DefaultForwarder) SubmitSeries(payload Payloads, extra http.Header) error {
+	transactions := f.createHTTPTransactions(endpoints.SeriesEndpoint, payload, true, extra)
```

Comment:
You can set `apiKeyInQueryString` to false, the API key is only looked for in the headers on this endpoint.